### DISCUSSION TITLE
Unable to pass verification because First name is covered by University Stamp

### DISCUSSION THREAD

**Discussion by dragor-radu** [community/community#119946] [GitHub Education]
### Hi everyone,

My first name on the University ID is covered by the University Stamp and I get the response that First name is not the same as in billing information and I am unable to solve this problem, I tried in many ways but no chance to solve this issue.

What should I try to do?

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
